website/docs: Update index.md for Sonarr (#15395)
goauthentik/authentik: website/integrations/services/sonarr/index.md

sonarr_password: password
```

> Additional note:  Make sure the type of authentication is set to "Basic (Browser Popup)" in Sonarr, otherwise the credentials don't ge passed.

Reviewer: ```suggestion
Additional note:  Make sure the type of authentication is set to "Basic (Browser Popup)" in Sonarr, otherwise the credentials are not passed.
```
